content/tutorials/2.projects/use-directus-templates-in-clay-for-data-enrichment.md

```diff
@@ -0,0 +1,417 @@
+| `_eq`        | Equals                      | `filter[status][_eq]` → `published`       |
+| `_neq`       | Not equals                  | `filter[status][_neq]` → `draft`          |
+| `_contains`  | Contains (case sensitive)   | `filter[title][_contains]` → `Guide`      |
+| `_icontains` | Contains (case insensitive) | `filter[title][_icontains]` → `guide`     |
+| `_in`        | In array                    | `filter[status][_in]` → `draft,published` |
+| `_gt`        | Greater than                | `filter[views][_gt]` → `1000`             |
+| `_lt`        | Less than                   | `filter[price][_lt]` → `100`              |
+| `_null`      | Is null                     | `filter[deleted_at][_null]` → `true`      |
+| `_nnull`     | Is not null                 | `filter[published_at][_nnull]` → `true`   |
+
+### Selecting Specific Fields
+
+To improve performance and reduce data transfer, specify only the fields you need:
+
+**In query parameters:**
+
+- **Key:** `fields`
+- **Value:** Comma-separated field names (e.g., `id,title,status,author`)
+
+**Including related fields:**
+
+- Use dot notation: `author.first_name,author.last_name`
+- This pulls in data from related collections
+
+### Sorting and Pagination
+
+**Sorting:**
+
+- **Key:** `sort`
+- **Value:** Field name (prefix with `-` for descending)
+- Example: `-date_created` (newest first)
+
+**Pagination:**
+
+- **Key:** `limit` - Maximum items to return (e.g., `50`)
+- **Key:** `offset` - Number of items to skip (e.g., `0`, `50`, `100`)
+
+---
+
+## Common Use Cases
+
+### E-commerce Product Enrichment
+
+**Scenario:** Enrich product data with inventory and pricing information
+
+1. Use **Get Item from Collection** to check if a product exists in Directus
+2. Use **Update Item in Collection** to update stock levels from external sources
+3. Use **Get Related Item Details** to pull supplier information
+
+### Content Publication Workflow
+
+**Scenario:** Automatically publish content when it's approved in Clay
+
+1. Use Directus webhooks to send draft content to Clay for review
+2. Enrich content with SEO metadata and keyword research in Clay
+3. Use **Update Item in Collection** to publish content back to Directus
+
+### Lead Enrichment System
+
+**Scenario:** Sync CRM data between Clay and Directus
+
+1. Use Directus webhooks to send new leads to Clay
+2. Enrich leads with company data and contact information
+3. Use **Update Item in Collection** to sync enriched data back
+4. Use **Get Related Item Details** to pull company profiles
+
+### Form Submission Processing
+
+**Scenario:** Process form submissions and create records
+
+1. Use Directus webhooks to send form submissions to Clay
+2. Use **Get Item from Collection** to check for existing records
+3. Use **Create Item in Collection** to add new contacts
+4. Use **Update Item in Collection** to update existing records
+
+---
+
+## Best Practices
```

Comment:
Wondering the Best Practices section is needed or if we can keep it more specific to Clay, and link to other places for stuff like Auth, Performance, etc.